src/helpers/PdfDownload/index.js

```diff
@@ -0,0 +1,38 @@
+export class PdfDownload {
+  constructor(event, form) {
+    this.submitEvt = event;
+    this.formInstance = form;
+  }
+
+  /**
+   * isPdfDownloadEnabled function check needPdf field is marked as 'yes' in form.io
+   * and then set PDF url as a session storage
+   * @return {undefined}
+   * */
+  isPdfDownloadEnabled() {
+    if (this.submitEvt.data && this.submitEvt.data.needPdf === "yes") {
+      const pdfUrl = this.submitEvt.metadata.pdfUrl.DownloadUrl;
```

Comment:
i assume the user will need to create a form action `pdfUrl` in their form settings? also the user will need to create a hidden field to store `needPdf` in the form?

Reply:
need pdf is an option that will be there in the form, so if a user wants that then they will check that

Reply:
> unsure why id is used instead of class, an can you do this without jquery?

I am not using jQuery in any code, the jquery is included by @devonpis I am not sure if that is to make it work with other forms?

Reply:
there is a little bit of legacy code still depending on jquery, unless we clean up those code first, not related to this ticket tho